gpflow/kullback_leiblers.py

```diff
@@ -22,7 +22,7 @@
 
 
 @name_scope()
-def gauss_kl(q_mu, q_sqrt, K=None):
+def gauss_kl(q_mu, q_sqrt, K=None, *, K_cholesky=None):
```

Comment:
Huh, does this work?? I had no idea! so you can pass K both by position or keyword, but K_cholesky must be passed as a keyword argument?

Reply:
Yeah, it works as you would expect. 
```
In [6]: def foo(a=None, *, b=None):
   ...:     print(a, b)

In [7]: foo(a=1, b=2)
1 2

In [8]: foo(a=1)
1 None

In [9]: foo(b=2)
None 2

In [10]: foo(1, b=2)
1 2

In [11]: foo(2, a=1)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-11-00078d28f012> in <module>
----> 1 foo(2, a=1)
TypeError: foo() got multiple values for argument 'a'

In [12]: foo(1, 2)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-12-d3151bdab612> in <module>
----> 1 foo(1, 2)
TypeError: foo() takes from 0 to 1 positional arguments but 2 were given
```

I think it's the best thing to do as it is backwards compatible and "forces" people to explicitly specify if they passing the cholesky

Reply:
Yeah :) I think it's great!

Reply:
(I just had no idea it would work)